[FAQ doc]How to load outside jar to classpath (#6273)
apache/skywalking: docs/en/FAQ/README.md

* [Compatible with other javaagent bytecode processing](Compatible-with-other-javaagent-bytecode-processing.md)
* [**Java agent memory leak** when enhance `Worker thread` at use Thread Pool](Memory-leak-enhance-Worker-thread.md)
* [Thrift plugin](thrift-plugin.md)

> * ["java -ext.dirs" is not supported,So how to load outside jar to classpath?](Load-outside-jar-to-classpath.md)

Reviewer: ```suggestion
* [How to resolve the issue, "java -ext.dirs" is not supported](Load-outside-jar-to-classpath.md)
```

Please change the name of markdown file to `resolve-ext-dir-issue.md`.
